### ISSUE TITLE
To improve accuracy in my own dataset from WordClassification DM

### ISSUE THREAD

**Issue by yupmanojhere** [Breta01/handwriting-ocr#57]
Thanks Bret,
It solved my error.
I got Another Question,
I have been training CharClassifier of CNN model,
I came across Train_steps,Test_iteration,save_iteration, and Batch Size
while I'm giving giving default value for training for 200 word images,and train_steps of 1000 it gives accuracy of 80.
But i need Accuracy above 95,in which OCR have to predict correct output.
may i know the Correct way for that.

**Comment by Breta01** [comment]
Firs consider increasing `train_steps`, so that the (green line) training accuracy converges to the 100%. Other mentioned variables shouldn't effect the result. Then you can improve the accuracy by increasing regularization (so the model doesn't overfit that much) or extending the training dataset.

**Comment by yupmanojhere** [comment]
Thanks  Breta,
i increased train_steps and i also got 92% accuracy,But At Final,While Using That model in OCR word prediction is 70-75%.
I also Trained GapClassification with my dataset.
Still final accuracy in OCR is not upto 70%.
help me out Breta.

**Comment by Breta01** [comment]
If the accuracy didn't changed, make sure you are using the models that you trained. Char classifier location is specified in [OCR.ipynb](https://github.com/Breta01/handwriting-ocr/blob/master/OCR.ipynb) as:
`MODEL_LOC = 'models/char-clas/' + LANG + '/CharClassifier'`
The gap classifier is loaded in [ocr/charSeg.py](https://github.com/Breta01/handwriting-ocr/blob/master/ocr/charSeg.py). `segmentation()` function is called from from [OCR.ipynb](https://github.com/Breta01/handwriting-ocr/blob/master/OCR.ipynb) as follows, so make sure you are using correct model (simple CNN vs RNN):
`gaps = charSeg.segmentation(img, RNN=True, debug=True)`

**Comment by MrOCR** [comment]
@Breta01 I would recommend that you have a look at this project:
http://visual.cs.ucl.ac.uk/pubs/handwriting/
https://github.com/thaines/helit/tree/master/handwriting

One of the problems that you are currently facing is due to the limited training dataset.
Thus, you can generate & synthesize text using the project that I suggested to you.

**Comment by Breta01** [comment]
@MrOCR Thanks for the tip! I will definitely take a look at it.

I am aware of this issue and in the recent rework, I added new datasets, but I haven't retrained the models yet.
